### PR TITLE
Use general JsonDict instead of specialized classes

### PR DESCRIPTION
**Pull request by SquishyStrawberry**
Pretty much what it says on the tin.

Comment with :+1: (: +1 : without the spaces) if you're fine with the merge and comment any issues you might have.

### CODE REVIEW

File: pythonwars/util.py [hunk 1]
```diff
@@ -1,28 +1,32 @@
 #!/usr/bin/env python3
-import sys
-
 import inflection
 
-__all__ = ["AttributesFromJson"]
-
+__all__ = ["JsonDict"]
```
Comment (raylu) [inline pythonwars/util.py]: we're not doing `import *` anywhere any more

File: pythonwars/util.py [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
-                module_name = ".".join(module_name)
-                result = sys.modules[module_name].__dict__[class_name](result)
-            setattr(self, inflection.underscore(leaf), result)
+        self._data = json_data
+
+    def __getitem__(self, key):
+        value = self._data[inflection.camelize(key, False)]
+        # We only want to convert "real" dicts
+        if value.__class__ is dict:
+            value = self.__class__(value)
```
Comment (raylu) [inline pythonwars/util.py]: should probably convert to snake_case and build inner classes when we read initially

File: pythonwars/util.py [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
+        return value
+
+    def __setitem__(self, key, value):
+        self._data[inflection.camelize(key, False)] = value
```
Comment (raylu) [inline pythonwars/util.py]: is this a supported operation? I don't think there are any uses cases for it yet

if there are, we need setattr too

**Comment by raylu** [comment]
why are we supporting getattr and getitem? let's pick exactly one, please

File: pythonwars/util.py [hunk 4]
```diff
@@ -1,28 +1,32 @@
 #!/usr/bin/env python3
```
Comment (raylu) [inline pythonwars/util.py]: I'm still not generally sold on having files named "util". can we move this class where it belongs (pythonwars.py or jsondict.py)?

**Comment by theelous3** [comment]
I'm happy with this

**Pull request merged**